src/main/scala/dx/core/languages/Language.scala

```diff
@@ -1,11 +1,12 @@
 package dx.core.languages
 
+import org.w3id.cwl.cwl1_2.CWLVersion
 import wdlTools.syntax.WdlVersion
 import dx.util.Enum
 
 object Language extends Enum {
   type Language = Value
-  val WdlVDraft2, WdlV1_0, WdlV2_0, CwlV1_2 = Value
+  val WdlVDraft2, WdlV1_0, WdlV2_0, CwlV1_0, CwlV1_1, CwlV1_2 = Value
```

Comment:
This is not necessary - we will only support v1.2. Earlier versions must be upgraded via a pre-processing step. Going forward, we'll have to decide if we want to natively support additional versions, or only support the latest version and always require earlier versions to be upgraded.